src/Library/demos/Launcher/main.blp

```diff
@@ -23,73 +23,72 @@ Adw.StatusPage {
         Gtk.StackPage {
           name: "file_launcher";
           title: _("File Launcher");
-          child:
+          child: Box {
```

Comment:
this is in our style guide :wink: 

https://github.com/sonnyp/Workbench/wiki/Style-Guide